mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.DeleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
+      if (error) {
+        return res.status(200).json({ 
+          success: "Order deletion failed", 
```

Comment:
In the DeleteOrder function, when an error occurs, the status code should be 400 instead of 200. The response should also include an error message. 

Here is the corrected code:
```
exports.DeleteOrder = (req, res) => {
  Order.remove({ _id: req.params.orderId }) 
    .exec((error, result) => {
      if (error) {
        return res.status(400).json({ 
          error: "Order deletion failed" 
        });
      }
      res.json("Order successfully deleted");
    });
};
```